src/config.rs

```diff
@@ -174,36 +174,31 @@ impl Config {
     }
 
     /// Reads all the apk files in the downloads folder and adds them to the configuration.
-    fn read_apks(&mut self) {
-        match fs::read_dir(&self.downloads_folder) {
-            Ok(iter) => {
-                for entry in iter {
-                    match entry {
-                        Ok(entry) => {
-                            if let Some(ext) = entry.path().extension() {
-                                if ext == "apk" {
-                                    self.add_app_package(entry.path()
-                                        .file_stem()
-                                        .unwrap()
-                                        .to_string_lossy()
-                                        .into_owned())
-                                }
-                            }
-                        }
-                        Err(e) => {
-                            print_warning(format!("There was an error when reading the \
-                                                   downloads folder: {}",
-                                                  e.description()));
+    fn read_apks(&mut self) -> Result<()> {
+        let iter = fs::read_dir(&self.downloads_folder)?;
+
+        for entry in iter {
+            match entry {
+                Ok(entry) => {
+                    if let Some(ext) = entry.path().extension() {
+                        if ext == "apk" {
+                            self.add_app_package(entry.path()
+                                .file_stem()
+                                .unwrap()
+                                .to_string_lossy()
+                                .into_owned())
                         }
                     }
                 }
-            }
-            Err(e) => {
-                print_error(format!("There was an error when reading the downloads folder: {}",
-                                    e.description()));
-                exit(Error::from(e).into());
+                Err(e) => {
+                    print_warning(format!("There was an error when reading the \
```

Comment:
Why is this now a warning instead of an error? If there is no downloads folder seems pretty bad, the analysis won't be performed.